### DISCUSSION TITLE
Turbo UI: add `j` and `k` to navigate task list

### DISCUSSION THREAD

**Discussion by DerTimonius** [vercel/turborepo#8377] [Ideas]
### Goals

1. Add additional keys to move through the task list other than the arrow keys
2. Maybe also add `h` and `l` to interact with the running task alongside `Enter` and `Ctrl Z`

### Background

While the arrow keys are a great way to navigate the task list in the new TUI, a lot of CLI tools use the vim typical navigation with `j` for down and `k` for up, for example `git diff`, `man` or `bat`.
Therefore, my first instinct when I wanted to navigate the list was to use j and k.

Additionally, a lot of developers, myself included, try to reduce the amount of time they have to move their hands away from the homerow of the keyboard. Using the arrow keys makes it necessary to move to a different part of the keyboard.

### Proposal

As this should be a pretty simple feature, I am very much interested in contributing, but wanted to get some feedback if this is even an issue for anyone but me 🙂

**Comment by anthonyshew** [comment]
Closing as duplicate of #8313.
